assets/src/scss/00-base/_pikaday.scss

```diff
@@ -1,5 +1,11 @@
 //pikaday is a JS plugin for date inputs
 
+@use "variables" as *;

```

Comment:
to simplify this should we just `@use "global" as *`; since everything in global.scss is either mixins or variables